mskokova/05_HTML/task_01.html

```diff
@@ -0,0 +1,164 @@
+							<INPUT TYPE="radio"value="2"NAME="radio">неплохо
+							</LABEL><br>
+						<LABEL>
+							<INPUT TYPE="radio"value="3"NAME="radio">хорошо
+							</LABEL><br>
+						<LABEL>
+							<INPUT TYPE="radio"value="4"NAME="radio">супер
+							</LABEL><br>
+						<input type="submit" value="Отправить"><br>
+						<input type ="hidden" name="date_voite" value="2013.12.31">
+					</FIELDSET>
+				</FORM>
+			</td></tr>
+			<tr><td><STRONG>Отзыв:</STRONG></td>
+			<td><STRONG>Выпадающий и мульти списки:</STRONG></td></tr>
+			<tr><td>
+				<FORM method="post" action="http://sc.it-academy.by/fe/form.php">
+					<FIELDSET>
+						<LEGEND>Оставьте Ваш отзыв</LEGEND>
+						<INPUT TYPE="text" ID="name" NAME="name" required>
+						<label for="name">* Имя</label><br>
+						<INPUT TYPE="email" ID="email" NAME="email" required>
+						<label for="email">* e-mail</label><br>
+						<textarea name="text" lang="ru">Форма отправки</textarea>
+						<input type="submit" value="Отправить"><br>
+					</FIELDSET>
+				</FORM>
+			</td>
+			<td>
+				<FORM method="post" action="http://sc.it-academy.by/fe/form.php">
+					<FIELDSET>
+						<LEGEND>Города РБ</LEGEND>
+						<select name="lis" id="list">
+							<optgroup label="Города РБ">
+								<option value="1">Брест</option>
+								<option value="2">Минск</option>
+								<option value="3">Витебск</option>
+							</optgroup>
+							<optgroup label="Города России">
+								<option value="4">Москва</option>
+								<option value="5">Рязань</option>
+								<option value="6">Магадан</option>
+							</optgroup>
+						</select>
+						<select name="list[]" id="list2" multiple size="8">
+							<optgroup label="Города РБ">
+								<option value="7">Брест</option>
+								<option value="8">Минск</option>
+								<option value="9">Витебск</option>
+							</optgroup>
+							<optgroup label="Города России">
+								<option value="10">Москва</option>
+								<option value="11">Рязань</option>
+								<option value="12">Магадан</option>
+							</optgroup>
+						</select>
+						<input type="submit" value="Ok">
+					</FIELDSET>
+				</FORM>
+			</td></tr>
+			<tr><td><STRONG>Загрузка файла на сервер:</STRONG></td>
+			<td><STRONG>Валидация данных HTML5:</STRONG></td></tr>
+			<tr><td>
+				<FORM method="post"enctype="multipart/form-data"action="http://sc.it-academy.by/fe/form.php">
+					<FIELDSET>
+						<LEGEND>Upload</LEGEND>
+						Файл:<br>
+						<input type="file" name="file" dir="ltr"><br>
+						Срок хранения (года):<br>
+						<INPUT type="text"name="time">
+						<input type="submit" value="загрузить">
+					</FIELDSET>
+				</FORM>
+			</td>
+			<td>
+				<FORM method="post" action="http://sc.it-academy.by/fe/form.php">
+					<FIELDSET>	
+						<LEGEND>Строка поиска</LEGEND>
+						<INPUT TYPE="text"  NAME="text" placeholder="Только русские буквы"lang="ru"><br>
+						<INPUT TYPE="text" ID="index" NAME="index"
+						pattern="\+375-(29|44|25)-[0-9]{7}"placeholder="+375-29-1234567">
```

Comment:
Wrong pattern

Reply:
Thanks. Do you mean should be without "-"? Like +375(29|44|25)[0-9]{7}? 
Cause I misunderstand the task.